tensorflow/python/kernel_tests/BUILD

```diff
@@ -2351,7 +2351,7 @@ cuda_py_test(
 
 cuda_py_test(
     name = "map_stage_op_test",
-    size = "small",
+    size = "medium",
```

Comment:
This is actually unneeded, but the bug causing the stage_op_test to occasionally hang does need to be fixed.  And verify if it affects the MapStagingArea as well.